src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts

```diff
@@ -669,39 +669,45 @@ export const walkthroughs: GettingStartedWalkthroughContent = [
 	},
 	{
 		id: 'NewWelcomeExperience',
-		title: localize('gettingStarted.setup.title', "Get Started with VS Code"),
-		description: localize('gettingStarted.setup.description', "Customize your editor, learn the basics, and start coding"),
+		title: localize('gettingStarted.new.title', "Get started with VS Code"),
+		description: localize('gettingStarted.new.description', "Supercharge coding with AI"),
 		isFeatured: false,
 		icon: setupIcon,
 		when: '!isWeb',
-		walkthroughPageTitle: localize('gettingStarted.setup.walkthroughPageTitle', 'Setup VS Code'),
+		walkthroughPageTitle: localize('gettingStarted.new.walkthroughPageTitle', 'Set up VS Code'),
 		content: {
 			type: 'steps',
 			steps: [
 				{
 					id: 'copilotSetup.chat',
-					title: localize('gettingStarted.agentMode.title', "Agent Mode"),
-					description: localize('gettingStarted.agentMode.description', "Tackle complex, multi-step tasks with AI"),
+					title: localize('gettingStarted.agentMode.title', "Agent mode"),
+					description: localize('gettingStarted.agentMode.description', "From idea to implementation, agent mode reasons through the problem, plans next steps, and applies changes"),
```

Comment:
"Reasons through the problem, plans next steps, and applies changes"
"Autonomously plans, orchestrates, and implements complex tasks"

Reply:
Oooh I like "Reasons through the problem, plans next steps, and applies changes"